app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -0,0 +1,19 @@
+$govuk-cookie-banner-background: #d5e8f3;
+
+.js-enabled {
+  .gem-c-cookie-banner {
+    display: none; // shown with JS, always on for non-JS
+  }
+}
+
+.gem-c-cookie-banner {
+  @include govuk-font($size: 16);
+  width: 100%;
```

Comment:
Setting width seems unnecessary for a block level element, or does it behave differently when in position?

Reply:
Yep, carried over from the other repo. Good spot.